compiler/scripts/pin_graphql_build_version.sh

```diff
@@ -0,0 +1,19 @@
+set -e
```

Comment:
please include this script in Dockerfile for dependecies predownload https://github.com/vmware-tanzu/graph-framework-for-microservices/blob/dddf7b52bc2f1a9e910febb66fd51dddf7090974/compiler/Dockerfile#L16-L16